data/mods/innawood/npcs/missiondef.json

```diff
@@ -0,0 +1,418 @@
+    },
+    "fail": { "effect": "npc_die" }
+  },
+  {
+    "id": "MISSION_GET_ANTIBIOTICS",
+    "type": "mission_definition",
+    "name": { "str": "Find Cattail Jelly" },
+    "goal": "MGOAL_CONDITION",
+    "difficulty": 2,
+    "value": 150000,
+    "urgent": true,
+    "goal_condition": { "u_has_item": "cattail_jelly" },
+    "origins": [ "ORIGIN_OPENER_NPC" ],
+    "deadline_low": 24,
+    "deadline_high": 48,
+    "dialogue": {
+      "describe": "This infection is bad, <very> bad…",
+      "offer": "I'm infected.  Badly.  I need you to get something antiseptic for me…",
+      "accepted": "Oh, thank god, thank you so much!  I won't last more than a couple of days, so hurry.  Get me some cattail jelly.  That should hopefully be good enough.",
+      "rejected": "What?!  Please, <ill_die> without your help!",
+      "advice": "I think if you can pull cattail jelly out of some cattails from like a swamp or something, I can smear it on the wound.",
+      "inquire": "Find any jelly yet?",
+      "success": "Oh thank god!  I'll be right as rain in no time.",
+      "success_lie": "What?!  You're lying, I can tell!  Ugh, forget it!",
+      "failure": "How am I not dead already?!"
+    },
+    "start": {
+      "effect": [
+        { "npc_add_effect": "infection", "duration": "PERMANENT" },
+        { "npc_remove_item_with": "cattail_jelly" },
+        "assign_guard",
+        { "clear_npc_rule": "investigate_noises" }
+      ]
+    },
+    "end": {
+      "effect": [ { "npc_lose_effect": "infection" }, "stop_guard", { "set_npc_rule": "investigate_noises" }, "npc_thankful" ]
+    },
+    "fail": { "effect": "npc_die" }
+  },
+  {
+    "id": "MISSION_GET_FLAG",
+    "type": "mission_definition",
+    "name": { "str": "Make a Flag" },
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 2,
+    "value": 100000,
+    "item": "grass_blanket",
+    "origins": [ "ORIGIN_OPENER_NPC", "ORIGIN_ANY_NPC" ],
+    "dialogue": {
+      "describe": "Does our flag still yet wave?  Maybe we will have to make it.",
+      "offer": "You haven't truly visited a place until you've driven a flagpole down into its dirt.  I've got the pole and the paint here, but I need a large canvas for the flag itself.  Get me a large sheet woven out of grass, and we can claim this land in the name of America!",
+      "accepted": "Hell ya!  Fetch me a large grass sheet!  Large enough to be used as a blanket!",
+      "rejected": "Seriously?  God damned commie…",
+      "advice": "You'll need to collect some straw and spin it into yarn first.",
+      "inquire": "Got the cloth?",
+      "success": "America, fuck ya!  I name this land…  New New England!  How does that sound?  I like it!",
+      "success_lie": "What?!  You liar!",
+      "failure": "You give up?  This country fell apart because no one could find a good man to rely on… might as well give up, I guess."
```

Comment:
```suggestion
      "failure": "You give up?  Our country fell apart because no one could find a good man to rely on, and now you're telling me this place ain't any better?  Might as well give up, I guess."
```